ansible/playbooks/srv/minidlna.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Looks like this is a real file? It should be a symlink to `../service/minidlna.yml` instead, so that the two playbooks don't get desynchronized in the future.